ui/app/rif/actions/index.js

```diff
@@ -678,27 +683,35 @@ function createPayment (partner, tokenAddress, netAmount, callbackHandlers = new
 function getChannels () {
   return (dispatch) => {
     return new Promise((resolve, reject) => {
-      background.rif.lumino.getChannels((error, channels) => {
-        if (error) {
-          dispatch(niftyActions.displayWarning(error));
-          return reject(error);
-        }
-        return resolve(channels);
-      });
+      if (rifConfig.mocksEnabled) {
```

Comment:
After the design refactor, this code will be deleted right? Maybe add a TODO here?